app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -66,9 +66,12 @@ def ems_inv_to_hashes
       # get_hosts
       get_images
       get_servers
+      # TODO: change spec to remove this
+      #
       get_volumes
```

Comment:
These lines - volumes, backups, snapshots - should be moved to the cinder_manager/refresh_parser.rb - which I think you already did - so these are redundant and should be removed.